### ISSUE TITLE
Bug: cannot rejoin existing household 

### ISSUE THREAD

**Issue by vincemue** [TomBursch/kitchenowl#158]
### Is this urgent?

Yes

### What parts are affected

Frontend

### What are the Server/Client versions of KitchenOwl

0.4.3

### What's the problem 🤔

After the Update I left my default household and was not able to rejoin this household again by myself, so that I had no access to my Data. Did I Miss anything?

Another User was able to add my again, but this could be a risk If you are the only User in the household.

**Comment by TomBursch** [comment]
Maybe the naming is not clear.
→ Leaving a household is the same as in leaving a messenger group (i.e. remove yourself from that group).
→ Switching the household is probably what you wanted to do.

Your data is not lost, though. Other members can add you back to the household, if that doesn't solve your issue, I will add an option to the management script of the backend and let you know how to add yourself back to the household later today.

**Comment by vincemue** [comment]
Thanks, fortunatly I Had another User that was able to add me again.

Maybe it would be an Option to add a warning before leaving or make the
Admin able to Put members into households via Account Administration later. In Case you are the only member of a household.

Thanks again.



Tom Bursch ***@***.***> schrieb am Do., 11. Mai 2023, 15:53:

**Comment by nr0dham** [comment]
I think I just ran into the same bug, got into a mode that gave me "no households, create one!" no matter what I did, web or Android.  I ended up hand-editing the SQL db with sqlite-browser to change the household on my account.  The default household from the upgrade was still in there.  Definitely not a great failure mode.

Oh, but so as not to leave things on a bad note, I just started using KitchenOwl and it's awesome!  Very high wife acceptance factor as well.  Very appreciative of your work!
